internal/endtoend/testdata/unsigned_params/mysql/go/query.sql.go

```diff
@@ -0,0 +1,19 @@
+// Code generated by sqlc. DO NOT EDIT.
+// versions:
+//   sqlc v1.19.1
+// source: query.sql
+
+package querytest
+
+import (
+	"context"
+)
+
+const createFoo = `-- name: CreateFoo :exec
+INSERT INTO foo (id) VALUES (?)
+`
+
+func (q *Queries) CreateFoo(ctx context.Context, id uint32) error {
```

Comment:
Note: The `id` parameter was previously generated as `int32` while it's unsigned in MySQL schema.